google/cloud/documentai_toolbox/wrappers/entity.py

```diff
@@ -26,44 +27,73 @@
+            If the entity is not present in
+            the document, this field will be empty.
+        start_page (int):
+            Required. `Page` containing the `Entity` or the first page of the
+            classification (for Splitter/Classifier processors).
+        end_page (int):
+            Required. Last page of the classification
+            (for Splitter/Classifier processors). Default is `0` for other processors.
+        normalized_vertices (List[documentai.NormalizedVertex]):
+            Optional. Vertices for bounding box of `Entity`.
     """
-    documentai_entity: documentai.Document.Entity = dataclasses.field(repr=False)
+
+    shard_index: int = dataclasses.field()
+    documentai_entity: dataclasses.InitVar[documentai.Document.Entity]
+
+    id: int = dataclasses.field(init=False)
     type_: str = dataclasses.field(init=False)
     mention_text: str = dataclasses.field(init=False, default="")
     normalized_text: str = dataclasses.field(init=False, default="")
 
-    # Only Populated for Splitter/Classifier Output
     start_page: int = dataclasses.field(init=False)
+    # Only Populated for Splitter/Classifier Output
     end_page: int = dataclasses.field(init=False)
 
-    # Only Populated for Identity Documents
-    image: Image.Image = dataclasses.field(init=False, default=None)
+    normalized_vertices: List[documentai.NormalizedVertex] = dataclasses.field(
+        init=False, default_factory=list
+    )
 
-    def __post_init__(self):
-        self.type_ = self.documentai_entity.type_
-        self.mention_text = self.documentai_entity.mention_text
+    def __post_init__(self, documentai_entity):
+        self.id = documentai_entity.id
+        self.type_ = documentai_entity.type_
+        self.mention_text = documentai_entity.mention_text
         if (
-            self.documentai_entity.normalized_value
-            and self.documentai_entity.normalized_value.text
+            documentai_entity.normalized_value
+            and documentai_entity.normalized_value.text
         ):
-            self.normalized_text = self.documentai_entity.normalized_value.text
+            self.normalized_text = documentai_entity.normalized_value.text
 
-        if self.documentai_entity.page_anchor.page_refs:
-            self.start_page = int(self.documentai_entity.page_anchor.page_refs[0].page)
-            self.end_page = int(self.documentai_entity.page_anchor.page_refs[-1].page)
+        page_refs = documentai_entity.page_anchor.page_refs
+        if page_refs:
+            self.start_page = int(page_refs[0].page)
+            self.end_page = int(page_refs[-1].page)
 
-    def crop_image(self, documentai_document: documentai.Document):
+            if page_refs[0].bounding_poly:
+                self.normalized_vertices = page_refs[
+                    0
+                ].bounding_poly.normalized_vertices
+
+    def crop_image(self, documentai_document: documentai.Document) -> Image.Image:
```

Comment:
Update the return type to `Optional[Image.Image]` since the method could return `None`.